bin/acala/cli/src/cli.rs

```diff
@@ -161,6 +161,12 @@ pub struct Cli {
 	/// Relaychain arguments
 	#[structopt(raw = true)]
 	pub relaychain_args: Vec<String>,
+
+	/// Instant block sealing
+	///
+	/// Can only be used with `--dev`
+	#[structopt(long = "instant-sealing")]
```

Comment:
```suggestion
	#[structopt(long = "instant-sealing", requires = "dev")]
```
https://docs.rs/structopt/0.2.13/i686-unknown-linux-gnu/structopt/clap/struct.Arg.html#method.requires